src/schemata/definitions/step/element.yml

```diff
@@ -5,16 +5,6 @@ description: |
   Elements are used to display data points over a certain period.
 required:
   - type
-oneOf:
```

Comment:
We have a problem here @kgrhartlage 
The way you fixed it on the previous commit to passing through the validation doesn't work for the laboperator. 
It invalidates at least four templates used to test the server.
How does it come up to throw this validation error "Unknown keyword 0" ?? 
There is no 0 in the JSON files.

Reply:
We passed the scope-creep threshold for this PR many commits ago 😵 

Which templates are invalid? I also struggle to reproduce this error.

The `inputChannels` schema appears to be wrong either way, since it currently expects an array or arrays. I wonder why that doesn't break right now on the server repo? 🤔

Reply:
[Elements.yml](https://github.com/labforward/laboperator-server/blob/develop/spec/support/files/workflow_templates/substeps/elements.yml)
[Markdown.yml](https://github.com/labforward/laboperator-server/blob/develop/spec/support/files/workflow_templates/markdown.yml)

Are examples. The problem seems to be related to inputField. When you let as you did before, and try to upload the template, it will return in the error message the fields containing errors.